ZAPD/OutputFormatter.cpp

```diff
@@ -1,8 +1,8 @@
 #include "OutputFormatter.h"
 
-int OutputFormatter::Write(const char* buf, int count)
+int32_t OutputFormatter::Write(const char* buf, int32_t count)
```

Comment:
The types here are supposed to be `int`s since this method will be used by libgfxd.